docs/source/guides/federated_testnet_guide.md

```diff
@@ -0,0 +1,283 @@
+# NuCypher Federated Testnet (NuFT) Setup Guide
+
+This guide is for individuals who intend to spin-up and maintain an Ursula node in the early stages of the NuFT 
+while working with the NuCypher team to improve user experience, comfort, and code-quality of the NuCypher network. 
+Following the steps will launch a functioning federated-only Ursula node operating from a machine you control. 
+Before getting started, please note: 
+
+* We encourage you to launch your node on a machine that is able to remain online with as few interruptions as possible, for as long as possible. Although it is possible to restart a node, excessive deactivation/reactivation can contribute to a lack of usable feedback from the test network. Additionally, node uptime will be important for proper functioning of Mainnet. Thus, it’s worthwhile to start working with a reliable machine now so that your nodes are compliant with our uptime requirements when the time comes.
```

Comment:
```suggestion
* We encourage you to launch your node on a machine that is able to remain online with as few interruptions as possible, for as long as possible. Although it is possible to restart a node, excessive deactivation/reactivation can contribute to a lack of usable feedback from the test network. Additionally, node uptime will be important for proper functioning of Mainnet. Thus, it’s worthwhile to start working with a reliable machine now so that your nodes are compliant with the network's uptime requirements when the time comes.
```